README.adoc

```diff
@@ -42,17 +42,17 @@ prebuilt definitions and move changes through environments via a visual interfac
 
 '''
 
-This repository centralizes issue tracking across Spinnaker for each microservice. The core code making up Spinnaker’s microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
-The core code making up Spinnaker’s microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
+This repository centralizes issue tracking across Spinnaker for each microservice. 
+The core code making up Spinnaker’s microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
 
 = Using Spinnaker
 
 Spinnaker users should refer to the main
 https://www.spinnaker.io/[Spinnaker site] and https://www.spinnaker.io/setup/[Installation] guide.
 
-For more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Documentation Overview].
+For more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Concepts] topic in the documentation Overview.
 
-If you're interested in a detailed walkthrough of the Spinnaker systems, check the https://www.spinnaker.io/reference/[Reference Material].
+If you're interested in a detailed walkthrough of the Spinnaker systems, check the https://spinnaker.io/guides/[User Guides] or the https://www.spinnaker.io/reference/[Reference Material].
```

Comment:
Good call adding the User Guides link.  

In my opinion, we can drop the link to the references page as there isn't much useful info there

Reply:
Agree. Done.